### PR TITLE
Revision ckr

### PR DESCRIPTION
**Pull request by swisstofu**

### CODE REVIEW

File: src/components/SearchField.jsx [hunk 1]
```diff
@@ -17,13 +17,13 @@ class SearchField extends Component {
...
     filterTokens(value);
-  }
+  };
 
   render() {
     const { onChangeSearch } = this;
     const { search } = this.props;
 
-  return (
+    return (
```
Comment (swisstofu) [inline src/components/SearchField.jsx]: formatting only

File: src/components/TokenForm.jsx [hunk 2]
```diff
@@ -1,12 +1,12 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import moment from 'moment';
-import { Form, Input, Button, Select } from 'antd';
-const { Option } = Select;
+import { Button, Form, Input, Select } from 'antd';
```
Comment (swisstofu) [inline src/components/TokenForm.jsx]: don't mix import and const

File: src/components/TokenForm.jsx [hunk 3]
```diff
@@ -44,10 +44,8 @@ class TokenForm extends Component {
 
   render() {
     const { getFieldDecorator } = this.props.form;
-    const { getCountriesOptions, handleSubmit } = this;
```
Comment (swisstofu) [inline src/components/TokenForm.jsx]: this extra code brings complexity without any added value

File: src/components/token-form.css [hunk 4]
```diff
@@ -1,4 +1,4 @@
 .token-form {
-  max-width: 500px;
+  max-width: 30rem;
```
Comment (swisstofu) [inline src/components/token-form.css]: try to avoid to mix em / rem and px

File: src/middleware/countries.js [hunk 5]
```diff
@@ -1,5 +1,5 @@
-const axios = require('axios');
-import { fetchCountries, setCountries, FETCH_COUNTRIES } from '../actions/countries';
+import axios from 'axios';
```
Comment (swisstofu) [inline src/middleware/countries.js]: avoid to mix require and import

**Review by swisstofu — COMMENTED** [review]
Hello ! 

Thank you for this work this is very good job. 

Despite some formatting issues that would prevent project to be built if we add a linter, your code is very well structured and formatted. 

Some comments: 

- typescript could have been used. it would bring some code completion help + display error directly in code editor (the pattern "fail fast" is my favorite... the earlier you get the error the better)
- country middleware: shouldn't you use next instead of store.dispatch to dispatch the setCountries action ?

**Review by Stafeeva — APPROVED** [review]

**Pull request merged**